### DISCUSSION TITLE
v1.81.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#4332] [Announcements]
# v1.81.0

## Highlights

Welcome to release `v1.81.0` of Immich. This release focuses on fixes for the recently discovered bug on the server.

There is no mobile release for this version.

- - - -

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

## What's Changed
* fix(server): delete face thumbnails when merging people by @danieldietzler in https://github.com/immich-app/immich/pull/4310
* feat(server): use host system timezone by @etnoy in https://github.com/immich-app/immich/pull/4313
* fix(server): incorrect video creation date EXIF extraction by @alextran1502 in https://github.com/immich-app/immich/pull/4309
* chore(server,web): bump node version to 20.8 by @etnoy in https://github.com/immich-app/immich/pull/4311
* fix(server): asset upload permissions for shared links by @danieldietzler in https://github.com/immich-app/immich/pull/4325
* fix(server): fallback to local timezone when rendering storage template by @jrasm91 in https://github.com/immich-app/immich/pull/4317
* chore(server): dev compose changes by @jrasm91 in https://github.com/immich-app/immich/pull/4316
* fix(server): library control doesn't apply to new library from the third row by @alextran1502 in https://github.com/immich-app/immich/pull/4331

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.80.0...

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.81.0'>v1.81.0</a>.</em>

**Comment by v3DJG6GL** [comment]
> fix(server): incorrect video creation date EXIF extraction by @alextran1502 in https://github.com/immich-app/immich/pull/4309

Is a "Extract Metadata" job enough to get the correct EXIF values in those cases? :)

**Reply by jrasm91** [reply]
You will have to do all, but yes

**Reply by v3DJG6GL** [reply]
great, thanks!
